scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -178,4 +215,36 @@ private String getBody() {
         }
         return String.format(BODY_TEMPLATE, jobID, status, allTaskStatusesString, hostname);
     }
+
+    private String getAttachment() throws NotConnectedException, UnknownJobException, PermissionException {
+        JobId jobID = jobState.getId();
+        logger.info("inside getAttachment, jobID: " + jobID.value());
+        List<TaskState> tasks = jobState.getTasks();
+        logger.info("inside getAttachment, before get job result");
+        JobResult result = service.getJobResult(jobID.value());
+        logger.info("inside getAttachment, after get job result");
+        String allRes = String.join(System.lineSeparator(),
+                                    tasks.stream()
+                                         .map(task -> result.getAllResults().get(task.getId()).getOutput().getAllLogs())
```

Comment:
it would be useful to add, before printing each task output, the task name and task id